### ISSUE TITLE
Fix WCAG reference in the help page for "scrollable-region-focusable"

### ISSUE THREAD

**Issue by iamrafan** [dequelabs/axe-core#1824]
Expectation: Map ```scrollable-region-focusable``` to WCAG 2.1.1 in the [help page](https://dequeuniversity.com/rules/axe/3.3/scrollable-region-focusable)

Actual: ```scrollable-region-focusable``` is mapped to WCAG 2.4.7

Motivation: Reference the rule to WCAG 2.1.1 which has the relevant failure criteria [F54](https://www.w3.org/TR/2016/NOTE-WCAG20-TECHS-20161007/F54)

<pre><code>axe-core version: 3.3.2-canary.1aa567c
axe-Coconut version: 3.9.0</code></pre>

**Comment by jeankaplansky** [comment]
Changed Reference the rule to WCAG 2.1.1 which has the relevant failure criteria https://www.w3.org/WAI/WCAG21/Techniques/failures/F55
